js/main.js

```diff
@@ -19,6 +19,94 @@ function submission(event) {
   formObject.entryId = data.nextEntryId;
   data.nextEntryId++;
   data.entries.unshift(formObject);
-  $image.src = 'images/placeholder-image-square.jpg';
+  $newImage.src = 'images/placeholder-image-square.jpg';
   $form.reset();
+  getEntry(data.entries[0]);
+  toggleNoEntries();
+  viewEntries();
+}
+
+function renderEntry(entry) {
+  var $entriesImg = document.getElementById('img' + entry.entryId);
+  var $entriesH2 = document.getElementById('h2' + entry.entryId);
+  var $entriesP = document.getElementById('p' + entry.entryId);
+  $entriesImg.src = entry.imageUrl;
+  $entriesH2.textContent = entry.title;
+  $entriesP.textContent = entry.notes;
+}
+
+document.addEventListener('DOMContentLoaded', getEntry('contentLoad'));
+
+function getEntry(entry) {
+  var length = 0;
+  if (entry === 'contentLoad') {
+    length = data.entries.length;
+    toggleNoEntries();
+  } else {
+    length = 1;
+  }
+  for (var i = 0; i < length; i++) {
+    var $ul = document.querySelector('ul');
+    var $li = document.createElement('li');
+    $li.setAttribute('class', 'row');
+    $ul.appendChild($li);
+
+    var $divImg = document.createElement('div');
+    $divImg.setAttribute('class', 'column-half');
+    $li.appendChild($divImg);
+
+    var $img = document.createElement('img');
+    var imgEntry = 'img' + data.entries[i].entryId;
+    $img.setAttribute('id', imgEntry);
+    $divImg.appendChild($img);
+
+    var $divText = document.createElement('div');
+    $divText.setAttribute('class', 'column-half');
+    $li.appendChild($divText);
+
+    var $h2 = document.createElement('h2');
+    var h2Entry = 'h2' + data.entries[i].entryId;
+    $h2.setAttribute('id', h2Entry);
+    $divText.appendChild($h2);
+
+    var $p = document.createElement('p');
+    var pEntry = 'p' + data.entries[i].entryId;
+    $p.setAttribute('id', pEntry);
+    $divText.appendChild($p);
```

Comment:
**_This_** is meant to be the `renderEntry` function. Your `renderEntry` function should be taking in a parameter which is a single object for a single entry, then `return`ing that DOM tree. The format will be **_very_** similar to what we did in `dom-creation`, you should review that exercise thoroughly.